server/models/bhima.sql

```diff
@@ -76,8 +76,8 @@ INSERT INTO unit VALUES
   (184, 'Job Title Management','TREE.PROFESSION','', 1,'/modules/functions/','/functions'),
   (185, 'Payroll Rubric Management','TREE.PAYROLL_RUB_MANAGEMENT','', 57,'/modules/payroll/rubrics','/payroll/rubrics'),
   (186, 'Holidays Management','TREE.HOLIDAYS_MANAGEMENT','Holidays Management',57,'/modules/holidays/','/holidays'),
-  (187, 'Offdays Management','TREE.OFFDAYS_MANAGEMENT','Offdays Management', 1,'/modules/offdays/','/offdays');
-
+  (187, 'Offdays Management','TREE.OFFDAYS_MANAGEMENT','Offdays Management', 1,'/modules/offdays/','/offdays'),
+  (188, 'Role management','TREE.ROLE_MANAGEMENT','Roles Management', 1,'/modules/role/','/role');
```

Comment:
All other routes are plural - `/invoices`, `/patients`, `/users`, etc...  Could you name this `/roles` to keep that pattern?